files/default/neutron-ha-tool.py

```diff
@@ -305,14 +304,15 @@ def run(args):
     return 1 if errors > 0 else 0
 
 
-def l3_agent_rebalance(qclient, noop=False, wait_for_router=True):
+def l3_agent_rebalance(qclient, args):
```

Comment:
> Did you checked this?

Yes.

> Also replacing from 3 to 2 arguments is not a big gain, and you hide the semantic of the signature.

Well, for ```l3_agent_evacuate``` it's reducing it from 8 to 5 (after the 2nd commit). I did this because the length of the argument lists was already raised in #7 and #8.
But lets see what other reviewers think. I am fine to rework it, if that the consensus.